test/val/val_primitives_test.cpp

```diff
@@ -28,13 +28,14 @@ using ValidatePrimitives = spvtest::ValidateBase<bool>;
 
 std::string GenerateShaderCode(
     const std::string& body,
+    const bool declare_geometry_streams_capability = true,
```

Comment:
This can be passed through capabilities_and_extensions.

Reply:
I've reworked it to use that parameter instead.  Good suggestion.